closure/goog/net/jsloader.js

```diff
@@ -111,8 +123,10 @@ goog.net.jsloader.loadMany = function(uris, opt_options) {
   if (isAnotherModuleLoading) {
     // jsloader is still loading some other scripts.
     // In order to prevent the race condition noted above, we just add
-    // these URIs to the end of the scripts' queue and return.
-    return;
+    // these URIs to the end of the scripts' queue and return the deferred
```

Comment:
OHHH. Haha, I thought "these" started a new sentence. A side effect of using the "unified" view. Sorry for confusion.